### ISSUE TITLE
Projectile size issue

### ISSUE THREAD

**Issue by Nexominer** [ThreeTAG/Palladium#100]
https://streamable.com/5djjdc

**Comment by Lucraft** [comment]
So, Pehkui scale doesnt apply to projectiles?

**Comment by Nexominer** [comment]
It does but its visual is delayed on summon and the hitbox does not since its custom set ig idk

**Comment by ThePhantomPig791** [comment]
isn't it supposed to be delayed because pehkui doesn't instantly set scale
try running /scale delay set 0 on it when it's first summoned 

i think i'm completely misunderstanding actually but 🤷

**Comment by Nexominer** [comment]
Either way if pehkui cant scale them on the first tick a command cant either...

**Comment by Nexominer** [comment]
Projectiles dont have a delay they just are shoot a bow for example

**Comment by ThePhantomPig791** [comment]
> Either way if pehkui cant scale them on the first tick a command cant either...

no, if you set the delay to 0 then it'll instantly scale and entity down

**Comment by Nexominer** [comment]
Ok but what im saying is throw a snowball they instant scale

**Comment by Nexominer** [comment]
Also the scale takes a single tick to kick in so by time the command kicks in it would cause the same effect

**Comment by Nexominer** [comment]
Projectiles like snowballs arrows potions ect do not have this issue
